packages/vite-plugin-angular/package.json

```
@@ -20,7 +20,8 @@
     "url": "https://github.com/analogjs/analog.git"
   },
   "peerDependencies": {
-    "@angular-devkit/build-angular": "^15.0.0 || >=16.0.0"
+    "@angular-devkit/build-angular": "^15.0.0 || >=16.0.0",
+    "@ngtools/webpack": "17.3.2"
```

I think we should match up the ranges

```suggestion
    "@ngtools/webpack": "^15.0.0 || >=16.0.0"
```